cli/ast/mod.rs

```diff
@@ -350,12 +350,23 @@ pub fn transpile_module(
   options: &EmitOptions,
   cm: Rc<SourceMap>,
 ) -> Result<(Rc<deno_ast::swc::common::SourceFile>, Module), AnyError> {
-  let source = strip_bom(source);
```

Comment:
I believe BOM should be stripped for JSON too, we do that in `deno_core`:https://github.com/denoland/deno/blob/de2df20d574b3a311d80d628cb7cefe20020d5e7/core/modules.rs#L783

Reply:
Yeah, I was thinking it was the "strip hashbang" logic for some reason, not the strip BOM stuff... 🤦